lib/rules_engine/set.rb

```diff
@@ -1,8 +1,11 @@
 class RulesEngine::Set
-  attr_reader :root, :name
+  attr_reader :name,
```

Comment:
Any reason for indenting down? It's not like we're developing using phone screens :P

Reply:
Looks beautiful man!

Reply:
lol

Reply:
But seriously, I changed it to match the format of `RulesEngine::Outcome` and `RulesEngine::Condition` plus I usually switch to a vertical format when the list gets to be 3 or 4 in length so that it makes it easier to scan the list for attributes. Being a converted trailing comma enthusiast, I'd love to hang a comma off the last item in the list but sadly I can't.